helm/reana/README.md

```diff
@@ -49,7 +49,8 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `notifications.enabled`                                  | Enable REANA system events notifications                                             | false                                           |
 | `notifications.system_status`                            | Cronjob pattern representing how often the system status notification should be sent. Leave it empty to deactivate it | "0 0 * * *"                                     |
 | `reana_url`                                              | REANA URL host                                                                       | None                                            |
-| `default_runtime_namespace`                              | Namespace in which the REANA runtime pods (workflow engines, jobs etc...) will run   | None                                            |
+| `runtime_namespace`                                      | Namespace in which the REANA runtime pods (workflow engines, jobs etc...) will run   | `.Release.Namespace`                            |
+| `dedicated_nodes`                                        | Should the infrastructure and runtime pods be split by node? (all nodes in the cluster should be labeled either as `reana.io/system=infrastructure` or `reana.io/system=runtime`)   | False |
```

Comment:
Tested:

- option off, no node labelled, expected pass -> OK

- option on, no node labelled, expect no deployment -> OK

- option on, one node labelled `reana.io/system=infrastructure`, expected deployment -> OK

- option on, one node labelled `reana.io/system=infrastructure`, no node labelled `reana.io/system=runtime`, expected workflow not to run -> but it did

- option on, one node labelled  `reana.io/system=infrastructure`, another node labelled ` `reana.io/system=runtime`, expected workflow to run on node two -> OK

So the feature work well overall, except for two comments:

(1) Unexpected success in the fourth case. Shouldn't we require some nodes to be labelled `reana.io/system=runtime` as per the docs? 

(2) BTW, instead of setting `dedicated_nodes: true` as a boolean variable to govern this behaviour, what about rather using two variables, something like:

```
node_label_infrastructure = reana.io/system=infrastructure
node_label_runtime = reana.io/system=runtime
```

so that people can use any labels they wish? E.g. they could be different, e.g. they could be equal, e.g. they could be "none" by default, which would switch this feature off. WDYT? Talk IRL.

Reply:
Applied, thanks for the suggestion.